stories/nav-drawer.stories.ts

```diff
@@ -13,11 +13,11 @@ const metadata = {
   component: 'igc-nav-drawer',
   argTypes: {
     position: {
-      type: '"start" | "end" | "top" | "bottom"',
+      type: '"start" | "end" | "top" | "bottom" | "relative"',
       description: 'The position of the drawer.',
-      options: ['start', 'end', 'top', 'bottom'],
+      options: ['start', 'end', 'top', 'bottom', 'relative'],
       control: {
-        type: 'inline-radio',
+        type: 'select',
```

Comment:
heh, that happens to me as well, might as well commit it if it keeps changing :)